app/models/framework/definition/RM3787.rb

```diff
@@ -20,7 +22,7 @@ class Invoice < Sheet
         field 'Unit of Purchase', :string, exports_to: 'UnitType'
         field 'Price per Unit', :decimal, exports_to: 'UnitPrice'
         field 'Quantity', :decimal, exports_to: 'UnitQuantity'
-        field 'Total Cost (ex VAT)', :decimal, exports_to: 'InvoiceTotal'
+        field 'Total Cost (ex VAT)', :decimal, exports_to: 'InvoiceValue'
```

Comment:
not for this PR as we want it in to get to the end of the export-rainbow, but this is making me start to think that the string identifiers for the export headers should be constants.

Reply:
Agree completely!

Reply:
(especially about the rainbow part)